pkg/puppet/puppet.go

```diff
@@ -31,6 +31,12 @@ type hieraData struct {
 	variables []string
 }
 
+var (
+	nvmeInstances = map[string]bool{
+		"c5d": true, "f1": true, "i2": true, "i3": true, "m5d": true, "r3": true, "r5d": true, "z1d": true,
```

Comment:
You missed some:
```
C5, C5d, i3.metal, M5, M5d, R5, R5d, T3, and z1d
```

Source: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html

Reply:
Good catch, I've promoted the logic to a function with some tests.